eng/build.ps1

```diff
@@ -573,6 +573,9 @@ function Deploy-VsixViaTool() {
   # Disable background download UI to avoid toasts
   &$vsRegEdit set "$vsDir" $hive HKCU "FeatureFlags\Setup\BackgroundDownload" Value dword 0
 
+  # Disable async save
+  &$vsRegEdit set "$vsDir" $hive HKCU "FeatureFlags\VS\RDT\AsyncSave" Value dword 0
```

Comment:
Do we need a bug to track this being enabled at some point? Although this is a good stop-gap, we at some point want to actually test our product in the way we ship, right?

Reply:
➡️ This is already removed